### PR TITLE
feature/EUI-4145-roles

### PR DESCRIPTION
**Pull request by rayliangatos**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4145

### Change description ###
The list of role categories to be presented can effectively be hard coded in the node layer, because it will always consist of 

Judicial
Legal ops
Admin

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by leepavlouhmcts — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
All looks good. I am familiar with the change because I've had to work around it in my current ticket

File: api/workAllocation2/index.ts [hunk 1]
```diff
@@ -302,3 +302,11 @@ export async function postTaskSearchForCompletable(req: EnhancedRequest, res: Re
     next(error);
   }
 }
+
+export async function getRolesCategory(req: EnhancedRequest, res: Response, next: NextFunction) {
+  const personRoles = [
+    {roleId: 'judicial', roleName: 'Judicial'},
+    {roleId: 'legalOps', roleName: 'Legal Ops'},
+    {roleId: 'admin', roleName: 'Admin'}];
```
Comment (udaydenduluri33) [inline api/workAllocation2/index.ts:310]: @rayliangatos - I remember @andywilkinshmcts / Aleena saying this is out of scope.

Comment (udaydenduluri33) [reply api/workAllocation2/index.ts:310]: Got an answer from Aleena. Admin is required. Thanks

**Review by udaydenduluri33 — APPROVED** [review]
I can see the role categories functionality implemented
